Src/AutoFixtureDocumentationTest/Extension/Constraints/ConstrainedStringGeneratorTest.cs

```diff
@@ -39,7 +39,7 @@ public void CreateAnonymousWillPrefixWithSeed()
             // Exercise system
             var result = sut.CreateaAnonymous(seed);
             // Verify outcome
-            Assert.True(result.StartsWith(seed));
+            Assert.StartsWith(seed, result);

```

Comment:
Thanks, but that's not me - those are XUnit [code analyzers](https://github.com/AutoFixture/AutoFixture/pull/773/commits/a451db59ca0708ea98cb346a4011307af701ed52) that are now present 😳